rust/bridge/node/futures/src/lib.rs

```diff
@@ -19,7 +19,7 @@
 //! [Neon]: https://neon-bindings.com/
 
 #![feature(wake_trait)]
-#![warn(missing_docs)]
+#![deny(missing_docs)]
```

Comment:
`warn` is deliberate, and actually I think we should standardize on it, because it lets you write code locally without docs. The libraries themselves should have zero warnings when compiled with their default toolchain.

Reply:
That sounds great, thanks! Done!